<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>890</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>890</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="paxlogo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/PAX.LABS.Wordmark.RGB.Black.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Step 1: Connect to and initialize the PSD4.</string>
    </property>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">height: 60px;
background-color: #fff;
border: 1px solid #333; 
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string>Search For Ports</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ports">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>381</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="initialize">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">height: 60px;
background-color: #fff;
border: 1px solid #333; 
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string>Initialize Pump</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Search for and select a port. Once the port has been selected</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>57</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>then initialize the pump.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>21</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PODOMATION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>56</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PSD4 Pump Controller</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Step 2: Speed and Acceleration Settings</string>
    </property>
    <widget class="QPushButton" name="set_speed">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">height: 60px;
background-color: #fff;
border: 1px solid #333; 
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string>Set Speed</string>
     </property>
    </widget>
    <widget class="QComboBox" name="speed">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>70</y>
       <width>187</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="set_accel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">height: 60px;
background-color: #fff;
border: 1px solid #333; 
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string>Set Acceleration</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Set speed and Aceleration Settings for the Pump</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="accel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>207</x>
       <y>70</y>
       <width>188</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>95</y>
      <width>841</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>401</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Manually Move the Pump</string>
    </property>
    <widget class="QPushButton" name="move_pump">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>106</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">height: 60px;
background-color: #fff;
border: 1px solid #333; 
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string>Move Pump</string>
     </property>
    </widget>
    <widget class="QSlider" name="position_slider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>6000</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>38</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Pump Position 0 &lt; x &lt; 6000.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="position">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>34</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>305</y>
      <width>841</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="initialized">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>327</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: tomato;
border-radius: 15px;
border: 1px solid #333;</string>
    </property>
    <property name="text">
     <string>Pump Offline</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speed_set">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>327</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ccc;
border-radius: 15px;
border: 1px solid #333;</string>
    </property>
    <property name="text">
     <string>Speed Not Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="accel_set">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>327</y>
      <width>139</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ccc;
border-radius: 15px;
border: 1px solid #333;</string>
    </property>
    <property name="text">
     <string>Accel Not Set</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="add_move">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 60px;
background-color: #fff;
border: 1px solid #333; 
border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_valve">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>370</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 60px;
background-color: #fff;
border: 1px solid #333; 
border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>Valve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_speed">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 60px;
background-color: #fff;
border: 1px solid #333; 
border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Command Builder</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="add_accel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>370</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 60px;
background-color: #fff;
border: 1px solid #333; 
border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>Accel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_delay">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>370</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 60px;
background-color: #fff;
border: 1px solid #333; 
border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>Delay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="execute_command">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>570</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 60px;
background-color: #fff;
border: 1px solid #333; 
border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QListWidget" name="command_list_widget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>410</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>paxlogo</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>line</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>line_2</zorder>
   <zorder>initialized</zorder>
   <zorder>speed_set</zorder>
   <zorder>accel_set</zorder>
   <zorder>line_3</zorder>
   <zorder>add_move</zorder>
   <zorder>add_valve</zorder>
   <zorder>add_speed</zorder>
   <zorder>label_7</zorder>
   <zorder>add_accel</zorder>
   <zorder>add_delay</zorder>
   <zorder>execute_command</zorder>
   <zorder>command_list_widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
